thorlcr/activities/keyedjoin/thkeyedjoinslave.cpp

```diff
@@ -894,7 +895,10 @@ class CKeyedJoinSlave : public CSlaveActivity, implements IJoinProcessor, implem
         void inc()
         {
             while (incNonBlocking())
+            {
+                BlockedActivityTimer t(activity.getTotalCyclesRef(), activity.queryTimeActivities());
```

Comment:
is this block time actually holding up the nextRow (or start) of the KJ act?
If it's not and overlapping time, e.g. because full, but nextRow is not blocked, then it would be wrong to consider it "blocked" time.
The time we are trying to track is the time the totalcycles was blocked.